internal/kernel/loader/catalog_test_helpers_test.go

```
@@ -0,0 +1,97 @@
+		if err != nil {
+			t.Fatal(err)
+		}
+		lock.Packages = append(lock.Packages, projectcontract.LockedPackage{
+			ID: manifest.ID, Version: manifest.Version, Source: "path:packages/" + manifest.ID,
+			ManifestSHA256: manifestSHA, LockSHA256: lockSHA,
+		})
+	}
+	return lock
+}
+
+func encodeProjectLock(t testing.TB, lock projectcontract.Lock) []byte {
```

golangci-lint (2.12.2)

[error] func encodeProjectLock is unused

(unused)